### DISCUSSION TITLE
DataGridXL3 is coming

### DISCUSSION THREAD

**Discussion by DataGridXL** [DataGridXL/DataGridXL2#97] [Announcements]
@manuelderuiter, @DataGridXL  Manuel, kun je me laten weten of je deze notification hebt ontvangen? (Sorry voor de spam!)

**Comment by manuelderuiter** [comment]
Die is ook binnen

**Comment by thdoan** [comment]
Wow, I just found out about DataGridXL2 and DataGridXL3 is coming? Exciting! What are some of the new features?
